cg/apps/hk.py

```diff
@@ -1,22 +1,26 @@
-# -*- coding: utf-8 -*-
+""" Module to decouple cg code from Housekeeper code """
```

Comment:
It seems that we have 81 files out of 230 in cg with this declaration. 81/230 is approx 35% so for consistency sake it would be nice to take a stand for or against having this declaration. 

According to this [Stackoverflow](https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3) post it's useless since Python 3 

I removed it since I don't see the point of having it here. What's your opinion on the matter? How do you do in Microsalt?

Reply:
I'll remove in all files